cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -575,7 +575,10 @@ func kubeClientGet(uc *UpstreamController, namespace string, name string, queryT
 		return nil, err
 	}
 	if err == nil && obj != nil {
-		util.SetMetaType(obj.(runtime.Object))
+		err := util.SetMetaType(obj.(runtime.Object))
+		if err != nil {
+			klog.Warningf("set MetaType error %v for obj %v, %v", err, queryType, obj.GetName())
+		}
```

Comment:
If the error in not null, we need to return err? cc @neiba

Reply:
i agree with you, the upper layer process guarantee that when err == nil, obj must be non-nil.

Reply:
already updated  @zc2638 @neiba @fisherxu